async_simple/Common.h

```diff
@@ -18,10 +18,15 @@
 
 #include <stdexcept>
 
-#define FL_LIKELY(x) __builtin_expect((x), 1)
-#define FL_UNLIKELY(x) __builtin_expect((x), 0)
+#ifdef _WIN32
+#define __builtin_expect(EXP, C) (EXP)
+#endif
 
-#define FL_INLINE __attribute__((__always_inline__)) inline
+#ifdef _WIN32
+#define AS_INLINE inline
```

Comment:
`inline` should have other meaning than suggesting a inlining. Let's define AS_INLINE as empty for windows.

Reply:
OK